specs/EnhancedSecurityMode.md

```diff
@@ -0,0 +1,468 @@
+        }
+                
+        MessageBox.Show("Enhanced Security Mode Enforce List is updated successfully",
+            "Enhanced Security Mode Enforce List", MessageBoxButtons.OK);
+    } 
+}
+```
+```cpp
+void SettingsComponent::GetEnhancedSecurityModeEnforceList() 
+{
+    wil::com_ptr<ICoreWebView2_13> webView2_13;
+    webView2_13 = m_webView.try_query<ICoreWebView2_13>();
+
+    if (webView2_13)
+    {
+        wil::com_ptr<ICoreWebView2Profile> profile;
+        CHECK_FAILURE(webView2_13->get_Profile(&profile));
+
+        auto profileStaging2 = profile.try_query<ICoreWebView2StagingProfile2>();
+        if (profileStaging2)
+        {
+            UINT32 counts = 0;
+            LPWSTR* uris;
+            profileStaging2->GetEnhancedSecurityModeEnforceList(&counts, &uris);
+            std::wstring s = L""; 
+            for (unsigned int i = 0; i < counts; ++i)
+            {
+               s += uris[i];
+               s += L"\n";
+            }
+
+            m_appWindow->AsyncMessageBox(s.c_str(), L"Enhanced Security Mode Enforce List");
+        }
+    }
+}
+
+void SettingsComponent::SetEnhancedSecurityModeEnforceList()
+{
+    wil::com_ptr<ICoreWebView2_13> webView2_13;
+    webView2_13 = m_webView.try_query<ICoreWebView2_13>();
+
+    if (webView2_13)
+    {
+        wil::com_ptr<ICoreWebView2Profile> profile;
+        CHECK_FAILURE(webView2_13->get_Profile(&profile));
+
+        auto profileStaging2 = profile.try_query<ICoreWebView2StagingProfile2>();
+        if (profileStaging2)
+        {
+            TextInputDialog dialog(
+                m_appWindow->GetMainWindow(), L"Add to Enhanced Security Mode Enforce List", L"List:",
+                L"Enter URI filters to enforce, separated by semicolons.", L"");
+            if (dialog.confirmed) {
+                // forming array
+                std::wstringstream ss(dialog.input);
+                std::wstring uri;
+                std::vector<std::wstring> uris;
+ 
+                while (std::getline(ss, uri, L';')) {
+                    uris.push_back(uri);
+                }
+ 
+                std::vector<LPCWSTR> uris_cstr;
+                for (const auto& uri : uris) {
+                    uris_cstr.push_back(uri.c_str());
+                }
+
+                CHECK_FAILURE(profileStaging2->SetEnhancedSecurityModeEnforceList(
+                    static_cast<uint32_t>(uris_cstr.size()), uris_cstr.data()));
+                MessageBox(nullptr, L"Enhanced Security Mode Enforce List is updated successfully",
+                    L"Enhanced Security Mode Enforce List", MB_OK);
+            }
+        }
+    }
+}
+```
+
+# API Details
+```
+/// Enhanced security mode levels
+[v1_enum]
+typedef enum COREWEBVIEW2_ENHANCED_SECURITY_MODE_LEVEL {
+  /// Enhanced security mode is turned off.
+  COREWEBVIEW2_ENHANCED_SECURITY_MODE_LEVEL_NONE,
+  /// The most restrictive level. This adds an extra layer of protection 
+  /// on all sites--familiar or unfamiliar.
```

Comment:
Familiar here is in reference to balanced which doesn't exist right? Should remove that text. Also, this enum value doesn't force applying on all sites - which sites this applies to depends on the other properties you set.